### DISCUSSION TITLE
how to start to configure the ATOM in order to get interaction with my GitHub-Account?

### DISCUSSION THREAD

**Discussion by tarifa10** [community/community#23296] [New to GitHub]
<p>dear github-experts</p>
<p>pretty new to github and Atom: </p>
<p>note; i am on Linux (Manjaro-Linux) at home; with ATOM and PyCharm</p>
<ul>
<li>and in office i have a windows-machine -  with Atom and PyCharm</li>
</ul>
<p><strong>the question i am working on: </strong> Version-Control with Atom: how to set &amp; configure the interaction of Atom and Github</p>
<p>i am pretty new to Atom and github - so i have a question: how to get up and configured the ATOM-editor so that it works with the Github-Account regarding the atom-manual on the following page i have several questions: <a href="https://flight-manual.atom.io/using-atom/sections/version-control-in-atom/" rel="nofollow noopener">https://flight-manual.atom.io/using-atom/sections/version-control-in-atom/</a></p>
<p>i think that for a beginner in ATOM - it is the question how to connect to the Github-Account. The below mentioned Atom-Manual notes how to work with the Git Account.  But the question of the day is : how to get to the</p>
<p><strong><span class="hashtag">#interaction</span> of Atom with the Github-account!?</strong><br>
that is the first question.</p>
<p>cf: <a href="https://flight-manual.atom.io/using-atom/sections/version-control-in-atom/" rel="nofollow noopener">flight-manual.atom.io/using-atom/sections/version-control-in-atom/</a></p>
<p>how to start to configure the ATOM in order to get interaction with my GitHub-Account?</p>
<p>love to hear from you</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>If you’re looking to use the Git features in Atom that are shown in the Flight Manual at the link you included, you first have to get Git working on your machine. In order to do that, you can <a href="https://help.github.com/articles/set-up-git/" rel="nofollow noopener">follow the instructions in the GitHub help documents</a>. Once Git is working on your machine and you’ve <a href="https://help.github.com/articles/create-a-repo/" rel="nofollow noopener">created a repository</a>, then you can <a href="https://help.github.com/articles/cloning-a-repository/" rel="nofollow noopener">clone that repository locally</a>. When that is all done, then you can open that local repository in Atom and use those features that you linked to in the Flight Manual. Atom reads the Git configuration that was created in each of the above steps and doesn’t need any additional setup to work.</p>
<p>I hope that helps!</p>

**Comment by tarifa10** [comment]
<p>hello dear Experts, </p>
<p>i have had some experience with ATOM and github - but now i have to do a restart. This thread helps me. </p>
<p>i have some questions to make clear that i understand all well…:  just set me straight if i am wrong.</p>
<p>we can merge GitHub with Atom, so you can pull your repo on GitHub to Atom,</p>
<p>that said we can:</p>
<ul>
<li>see changes on Atom,</li>
<li>make changes in Atom,</li>
<li>commit in Atom, and then push to GitHub, similiar to GitHub Desktop just with a built-in editor. the core function “Git-Integrator” is a great feature: </li>
</ul>
<p>i have started ATOM and i have logged into my GIthub-account - online. Now the question is: how to connect to github from ATOM - how to connect both!?  At the moment all looks like so:</p>
<p>We can add/commit/push from inside Atom and we can create new branches,<br>
note; i am a bit familiar with git?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/7aee445ea923fede30425ece0449200e66366f9e.png" data-download-href="/uploads/short-url/hxuJE5Og4FM0ynncpna7nC11fVc.png?dl=1" title="github_connection.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/45537527/181076069-5dc18506-968a-48bc-b7e0-1563cc1f0e43.png" alt="github_connection.png" data-base62-sha1="hxuJE5Og4FM0ynncpna7nC11fVc" width="690" height="294" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/7aee445ea923fede30425ece0449200e66366f9e_2_690x294.png, https://aws1.discourse-cdn.com/github/optimized/1X/7aee445ea923fede30425ece0449200e66366f9e_2_1035x441.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/7aee445ea923fede30425ece0449200e66366f9e_2_1380x588.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/7aee445ea923fede30425ece0449200e66366f9e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">github_connection.png</span><span class="informations">1448×619 97.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>love to hear from you </p>

**Comment by lee-dohm** [comment]
<p><a class="mention" href="/tarifa10">@tarifa10</a> I’m not sure what question you’re asking. Can you narrow things down to a single question or tell me exactly what task you’re trying to complete?</p>
<p>Thanks.</p>

**Comment by ncrowder** [comment]
<p>I think my problem is similar.  I created a file locally, used atom to create a local repository, even staged and comitted the changes within ATOM.  Great, but when I try the GITHUB panel and click on “publish on GitHub” the user field just keeps hanging and never populates.  It just says “loading” without end.  Is it only possible to start in Github and bring down to ATOM?  Why can’t I start in ATOM and connect to Github?</p>

**Comment by lee-dohm** [comment]
<p>Sounds like there’s a bug somewhere. Please go to <a href="https://github.com/atom/github/issues/new" rel="nofollow noopener">https://github.com/atom/github/issues/new</a> and file a new issue by completely filling out the required template.</p>

**Comment by ncrowder** [comment]
<p>Thank you.  I was wondering if I was going insane.  At least maybe I’m not.</p>
